nss-day-cohort-29/reactive-nutshell-copperden
# News articles

# Description
News articles appear at "/"
Buttons are not yet operational.

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout news-articles
3. "npm start" and run "json-server -p 5002 -w database.json"
4. Verify that news articles appear at localhost:3000

# Checklist:
